src/notify.js

```diff
@@ -71,6 +83,11 @@ Notify.isSupported = function(perm) {
         throw new Error('You must only call this before calling Notification.requestPermission(), otherwise this feature detect would trigger an actual notification!');
     }
 
+    if (Notify.serviceWorkerSupported()) {
```

Comment:
In Chrame Android, both buttons work right now. If you remove the `swUrl` option, the first button doesn't work.
In Safari, the first button works, the second doesn't.